README.md

```diff
@@ -1,9 +1,17 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/{APP-NAME}-app)
+[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/app-updater-app)
 
-# {APP-NAME}-app chart
+# app-updater-app chart
 
-Giant Swarm offers a {APP-NAME} Managed App which can be installed in tenant clusters.
-Here we define the {APP-NAME} chart with its templates and default configuration.
+Giant Swarm offers a App Updater which can be installed in clusters to update App Platform apps when there is a new version.
+
+It uses a simple bash script in a `ConfigMap` that pulls from Github the last release and compares with the version set in the App Custom Resource. If it is different then set the new version on the App CR. It runs by default every 30 minutes as `Cronjob` Kubernetes entity.
+
+There are two parameters to pass when the App Updater is going to be installed:
+
+```
+app_repo: docs
```

Comment:
This assumes that the repo belongs to the `giantswarm` org. Which means that the app only works for us. I wouldn't mind having to configure `giantswarm/docs` instead.

Reply:
good idea